packages/trader/src/Modules/CFD/Containers/cfd-password-modal.jsx

```diff
@@ -328,53 +328,75 @@ const CFDServerForm = ({ ...props }) => {
     }, [props.trading_servers]);
 
     return (
-        <Formik
-            initialValues={{
-                server: props.trading_servers.find(server => !server.disabled)?.id ?? '',
-            }}
-            validate={props.validateServer}
-            onSubmit={values => props.submitMt5Server(values.server)}
-        >
-            {({ handleSubmit, setFieldValue, errors, values, isSubmitting }) => (
-                <form onSubmit={handleSubmit}>
-                    <div className='cfd-password-modal__content'>
-                        <div className='dc-modal__container_cfd-password-modal__body'>
-                            <div className='input-element'>
-                                <RadioGroup
-                                    className='cfd-password-modal__radio'
-                                    name='server'
-                                    required
-                                    selected={props.trading_servers.find(server => !server.disabled)?.id}
-                                    onToggle={e => {
-                                        e.persist();
-                                        setFieldValue('server', e.target.value);
-                                    }}
-                                >
-                                    {available_servers.map(item => (
-                                        <RadioGroup.Item
-                                            key={item.value}
-                                            label={item.label}
-                                            value={item.value}
-                                            disabled={item.disabled}
-                                        />
-                                    ))}
-                                </RadioGroup>
+        <React.Fragment>
+            {props.has_error && (
+                <div className='cfd-password-modal__warning'>
+                    <Text
+                        as='p'
+                        className='cfd-password-modal__warning-text'
+                        lineHeight='l'
+                        size='xxs'
+                        color='prominent'
+                        weight='normal'
+                        align='center'
+                    >
+                        <Localize
+                            i18n_default_text='Due to an issue on our server, some of your {{platform}} accounts are unavailable at the moment. Please bear     with us and thank you for your patience.'
+                            values={{
+                                platform: props.platform === CFD_PLATFORMS.MT5 ? 'DMT5' : 'Deriv X',
+                            }}
+                        />
+                    </Text>
+                </div>
+            )}
```

Comment:
(suggestion) Can we refactor this into its own component? `<CFDPasswordModalWarning />` for example.